transforms/code/license_check/python/test-data/expected/metadata.json

```diff
@@ -0,0 +1,218 @@
+                "Zend-2.0",
+                "TCP-wrappers",
+                "XFree86-1.1",
+                "FSFUL",
+                "OLDAP-1.3",
+                "SGI-B-2.0",
+                "NetCDF",
+                "Zed",
+                "ZPL-2.0",
+                "Apache-1.0",
+                "CC-BY-1.0",
+                "OLDAP-2.1",
+                "OLDAP-1.2",
+                "OLDAP-2.0",
+                "NTP",
+                "AMPAS",
+                "Barr",
+                "mpich2",
+                "ANTLR-PD",
+                "Xerox",
+                "Spencer-94",
+                "AMDPLPA",
+                "BSD-3-Clause-No-Nuclear-License",
+                "HPND",
+                "ECL-1.0",
+                "MirOS",
+                "Qhull",
+                "ZPL-1.1",
+                "TU-Berlin-2.0",
+                "Spencer-86",
+                "SMLNJ",
+                "xinetd",
+                "OLDAP-2.2.2",
+                "MIT-enna",
+                "Font-exception-2.0",
+                "FSFULLR",
+                "TU-Berlin-1.0",
+                "xpp",
+                "NRL",
+                "W3C-19980720",
+                "EFL-1.0",
+                "eGenix",
+                "Unicode-DFS-2016",
+                "SWL",
+                "Spencer-99",
+                "Plexus",
+                "VSL-1.0",
+                "Leptonica",
+                "Unicode-DFS-2015",
+                "Mup",
+                "Giftware",
+                "OLDAP-2.2",
+                "APAFML",
+                "NBPL-1.0",
+                "OLDAP-1.1",
+                "Entessa",
+                "Multics",
+                "Newsletr",
+                "psutils",
+                "bzip2-1.0.5",
+                "Afmparse",
+                "diffmark",
+                "BSD-2-Clause-Views",
+                "DSDP",
+                "MIT-Modern-Variant",
+                "ANTLR-PD-fallback",
+                "Bahyph",
+                "BSD-3-Clause-Modification",
+                "BSD-4-Clause-Shortened",
+                "HTMLTIDY",
+                "MIT-open-group",
+                "MulanPSL-2.0",
+                "OLDAP-2.0.1",
+                "Saxpath",
+                "Borceux",
+                "Crossword",
+                "CrystalStacker",
+                "Rdisc",
+                "Wsuipa"
+            ],
+            "deny": false
+        },
+        "checkpointing": false,
+        "max_files": -1,
+        "random_samples": -1,
+        "files_to_use": [".parquet"]
+    },
+    "job_output_stats": {
+        "source_files": 2,
+        "source_size": 6991,
+        "result_files": 2,
+        "result_size": 7511,
+        "processing_time": 0.026211023330688477,
+        "source_doc_count": 4,
+        "result_doc_count": 4
+    },
+    "source": {
+        "name": "/Users/shivdeep/workspace/projects/current/oss-data-prep/data-prep-lab/transforms/code/license_check/python/test-data/input",
```

Comment:
here, directory location is hard coded. if test-framework is not going to compare all the keys, can we keep only the relevant keys in the metadata.json

Reply:
will see if it can be done